Probesys/bileto
# imp: Always put the "new" buttons on the right

## Related issue(s)



https://github.com/Probesys/bileto/issues/652

## How to test manually



- navigate across Bileto (tickets, contracts, organizations, admin)
- verify that the "New" buttons are always put on the right of the screen

## Checklist
